src/main/java/co/lotc/lever/cmd/Emote.java

```diff
@@ -0,0 +1,26 @@
+package co.lotc.lever.cmd;
+
+import co.lotc.core.bukkit.util.ChatBuilder;
+import co.lotc.core.bukkit.util.LocationUtil;
+import co.lotc.core.command.annotate.Arg;
+import co.lotc.core.command.annotate.Flag;
+import co.lotc.lever.BaseCommand;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import static org.bukkit.ChatColor.*;
+
+public class Emote extends BaseCommand {
+
+    @Flag(name="r",description="Emote broadcast radius",type=double.class)
```

Comment:
I'd prefer the radius be a default not a flag

Reply:
This is a good place to use flags I'd rather people learn to start using them becuase theyre much more informative in the long run.